bitnami/apache/templates/prometheusrules.yaml

```diff
@@ -8,19 +8,9 @@ apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   name: {{ include "common.names.fullname" . }}
-  {{- if .Values.metrics.prometheusRule.namespace }}
-  namespace: {{ .Values.metrics.prometheusRule.namespace }}
-  {{- else }}
-  namespace: {{ .Release.Namespace }}
-  {{- end }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    app.kubernetes.io/component: metrics
```

Comment:
You have accidentally removed this label. Please, add it again.

Reply:
done at https://github.com/bitnami/charts/pull/18291/commits/2b88dba2bb4d5b023b09787596e7fb31b8e5e5ff